seqr/views/utils/permissions_utils.py

```diff
@@ -23,6 +26,12 @@ def user_is_data_manager(user):
 def user_is_pm(user):
     return user.groups.filter(name=PM_USER_GROUP).exists() if PM_USER_GROUP else user.is_superuser
 
+def user_has_programmatic_access(user):
+    return user.groups.filter(name=PROGRAMMATIC_ACCESS_GROUP).exists()
+
+def user_is_active_and_has_programmatic_access(user):
```

Comment:
Why not always check whether the user is active?

Reply:
Because it didn't fit with the pattern above - and I'm not using their is_active implementation (because it redirects to login page)